torch/onnx/symbolic_helper.py

```diff
@@ -94,6 +97,7 @@
 ]
 
 
+@_beartype.beartype
 def _parse_arg(
```

Comment:
_parse_arg can return a lot of different types. Annotating the return type will make mypy mad at various callers because the return value can be any one of those specify. I suggest dealing with it in a separate change